### PR TITLE
Cart Drawer Implementation

### PR DESCRIPTION
**Pull request by rickycse**
Implemented the Figma design for the Cart Drawer

### CODE REVIEW

File: backend/src/main/java/com/MerchStore/backend/App/Driver.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.MerchStore.backend.App;
```
Comment (DROFEL) [inline backend/src/main/java/com/MerchStore/backend/App/Driver.java]: are you supposed to make any changes to backend?

**Review by namanrai — APPROVED** [review]

File: frontend/src/components/Product.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  DialogContent,
+  DialogTitle,
+  Typography,
+} from "@mui/material";
+import "../styles/Product.css";
+import React, { useState } from "react";
+
+export default function Product() {
+  return (
+    <Box className="product" color="background">
```
Comment (DROFEL) [inline frontend/src/components/Product.js]: These are legacy files can you remove this one and all css that are included in this merge

**Review by DROFEL — CHANGES_REQUESTED** [review]
Remove backend files commit

**Review by ShafinMahmudCS — CHANGES_REQUESTED** [review]
@Rickie457 You created this branch from an early version of development. You should have pulled the updates from the development branch. Many files were removed or renamed so in your pull request all of those are showing as new files added.

**Review by rickycse — COMMENTED** [review]
resolved conflicts

**Review by DROFEL — APPROVED** [review]
good

**Review by ShafinMahmudCS — APPROVED** [review]

**Pull request merged**